en_US/deploy/cluster/auto.md

```diff
@@ -49,7 +49,7 @@ cluster {
 }
 ```
 
-### DNS A records
+### `dns` (A records)
```

Comment:
Not sure if this is a better replacement

Reply:
I tried to make titles uniform — previously some records were identifiers, and some — regular phrases.

Reply:
✅